sechub-wrapper-prepare/src/test/java/com/mercedesbenz/sechub/wrapper/prepare/modules/skopeo/SkopeoWrapperTest.java

```diff
@@ -143,11 +143,10 @@ void when_process_throws_exception_then_download_throws_exception() throws IOExc
 
     }
 
-    private void assertProcessBuilderFactoryCalledWithParametersForLogin(SkopeoContext context, String location, String user,
-            ProcessBuilderFactory processBuilderFactory) {
+    private void assertProcessBuilderFactoryCalledWithParametersForLogin(String location, String user, ProcessBuilderFactory processBuilderFactory) {
 
         // skopeo, login, ubuntu:22.04, --username, username1, --password, password1,
-        // --authfile, authentication.json]
+        // --authfile, authentication.json
```

Comment:
Please add the --additional-tag part to the example output (or just remove the two example lines)

Reply:
skopeo login does not need additional tag and does not support additional tag